client/app/components/onboardingGuide/OnboardingGuideContainer.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import OnboardingGuide from './OnboardingGuide';
+
+import * as OnboardingGuideActions from '../../actions/OnboardingGuideActions';
+
+const OnbardingGuideContainer = ({ actions, data, railsContext }) => {
+  return (
+    <OnboardingGuide {...{actions, data, railsContext}} />
+  );
+}
+
+OnbardingGuideContainer.propTypes = {
+  actions: PropTypes.object.isRequired,
+  data: PropTypes.object.isRequired,
+  railsContext: PropTypes.object.isRequired
```

Comment:
At this point I added more specific to data, which is something we construct. 
RailsContext might change with gem updates and actions is